### DISCUSSION TITLE
X35A  & X50A connector type and wiring scheme?

### DISCUSSION THREAD

**Discussion by jujoparo** [revk/xESP32-Faikout#283] [Q&A]
Please, What is the connector type  of x35A and X50A?
And What is the wiring scheme?

Thank you

**Comment by jujoparo** [comment]
I have found the info here:

https://github.com/unixko/DaikinCon?tab=readme-ov-file

https://user-images.githubusercontent.com/44964969/62939597-d9cc5a00-bdfb-11e9-9e97-085322eea2e1.jpg

**Reply by 44616E** [reply]
Good find, was about to work out the connector myself.

**Reply by jujoparo** [reply]
You can buy it in Aliexpress for 10 euros in a set of 10 wires.

https://es.aliexpress.com/item/1005005465304996.html?spm=a2g0o.productlist.main.3.65c930a9w1XD9U&algo_pvid=bb2be4d9-83a9-4083-9bab-a4c89d8a6054&algo_exp_id=bb2be4d9-83a9-4083-9bab-a4c89d8a6054-1&pdp_npi=4%40dis%21EUR%2110.35%218.28%21%21%2110.87%218.70%21%40211b813f17128254456524668ec13d%2112000036432793231%21sea%21ES%210%21AB&curPageLogUid=2M9LwXJTorPA&utparam-url=scene%3Asearch%7Cquery_from%3A

**Reply by jaspweowaale** [reply]
Im not sure why, but they no add 50-100usd for shipping for 10usd USD item. The site will close soon, I feel

**Reply by revk** [reply]
I have launched a generic cable if it helps

I have it as an option on GitHub, but can probably do as a separate product if people want.

![2025-07-04T10-54-18 897Z-326073](https://github.com/user-attachments/assets/1907446d-3e12-4a4c-a37e-45375ef30d77)

**Reply by jaspweowaale** [reply]
Sure,  still need to get others parts, however what do you think cost is to send to Vietnam,  if need 5 or 10?

**Reply by revk** [reply]
I am trying to get listed now on Tindie for $8 plus postage. The idea is that for a lot of ACs the pin headers will work, but when not, the lead is half of what you need. I have a "rest of world" shipping set to $20, but I would have to check royal mail rates.

**Reply by jaspweowaale** [reply]
My main concern is the cost of the boards — I can purchase an ESP32 locally in Vietnam for around $5, but I’ve noticed that project-specific PCBs are significantly more expensive.
I’ve only recently started looking into this, so I’m not yet fully clear on the advantages and disadvantages of purchasing the official project PCB.
Additionally, we face a 45% import tax on items valued over $50 (including shipping).
Do you have any advice on how best to approach this  , in general I donate and buy from projects, because good Ideas need beer money

**Reply by revk** [reply]
That is a bigger. The boards have the logic to handle the 5V from the Daikin.

**Reply by jaspweowaale** [reply]
Yes, I like to buy already made, and support project dont mind the cost, but start to be a bit much when you add 45% and 10% VAT ,  but for smaller things like cable its under the 40usd tax limit and they might let it go

**Reply by revk** [reply]
https://www.tindie.com/products/revk/faikins21-generic-cable-50cm/
